packages/lib/src/encryption/Encryption.ts

```diff
@@ -272,10 +270,10 @@ export function checkSignature(
     }
 }
 
-export function signWithSignatureKey(message: Message, keys: Keys): string {
+export function signWithSignatureKey(message: any, keys: Keys): string {
     return ethers.utils.hexlify(
         nacl.sign.detached(
-            ethers.utils.toUtf8Bytes(JSON.stringify(message)),
+            ethers.utils.toUtf8Bytes(stringify(message)!),
```

Comment:
Can `stringify()` return something else than a string?

Reply:
It can return string | null | undefined